src/content/advanced/gitops/what-is-gitops/index.md

```diff
@@ -1,5 +1,5 @@
 ---
-linkTitle: What is GitOps
+linkTitle: What is it?
```

Comment:
[minor]

Rather keep "What is GitOps?" compared to "it" which isn't a self-explanatory word

Reply:
I did not want to repeat the parent section name. IMO it is self-explanatory since the parent section name is displayed in the doc entry 
![Screenshot 2023-12-11 at 14 20 33](https://github.com/giantswarm/docs/assets/892157/784e337d-0698-4991-a950-d269f40f735c)